docs/guides/admin/docs/releasenotes.md

```diff
@@ -51,6 +51,7 @@ Configuration changes
     - `etc/org.opencastproject.security.urlsigning.provider.impl.GenericUrlSigningProvider.cfg`
     - `etc/org.opencastproject.security.urlsigning.provider.impl.WowzaUrlSigningProvider.cfg`
     - `etc/org.opencastproject.security.urlsigning.verifier.impl.UrlSigningVerifierImpl.cfg`
+- Due [MH-13569](https://opencast.jira.com/browse/MH-13569), the variable in prop.player has been changed.
```

Comment:
Can you remove this line? This is nothing that has changed between releases and is only a modification within the release. From a release point of view `{{id}}` never existed.

Reply:
No problem, but where I left the comment of the change?

Reply:
You don't need to. The changelog is generated automatically.

Reply:
I see, Done!